opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -994,4 +994,232 @@ mod tests {
+        let instrumentation: InstrumentationLibrary = Default::default();
+
+        processor.emit(&mut record, &instrumentation);
+
+        assert_eq!(exporter.get_emitted_logs().unwrap().len(), 1);
+    }
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn test_simple_processor_sync_exporter_with_multi_thread_runtime() {
+        let exporter = InMemoryLogsExporterBuilder::default()
+            .keep_records_on_shutdown()
+            .build();
+        let processor = Arc::new(Mutex::new(SimpleLogProcessor::new(Box::new(
+            exporter.clone(),
+        ))));
+
+        let mut handles = vec![];
+        for _ in 0..10 {
+            let processor_clone = Arc::clone(&processor);
+            let handle = tokio::spawn(async move {
+                let mut record: LogRecord = Default::default();
+                let instrumentation: InstrumentationLibrary = Default::default();
+                processor_clone
+                    .lock()
+                    .unwrap()
+                    .emit(&mut record, &instrumentation);
+            });
+            handles.push(handle);
+        }
+
+        for handle in handles {
+            handle.await.unwrap();
+        }
+
+        assert_eq!(exporter.get_emitted_logs().unwrap().len(), 10);
+    }
+
+    #[tokio::test(flavor = "current_thread")]
+    async fn test_simple_processor_sync_exporter_with_current_thread_runtime() {
+        let exporter = InMemoryLogsExporterBuilder::default()
+            .keep_records_on_shutdown()
+            .build();
+        let processor = SimpleLogProcessor::new(Box::new(exporter.clone()));
+
+        let mut record: LogRecord = Default::default();
+        let instrumentation: InstrumentationLibrary = Default::default();
+
+        processor.emit(&mut record, &instrumentation);
+        processor.force_flush().unwrap();
+        processor.shutdown().unwrap();
+
+        assert_eq!(exporter.get_emitted_logs().unwrap().len(), 1);
+    }
+
+    #[derive(Debug, Clone)]
+    struct LogExporterThatRequiresTokioSpawn {
+        logs: Arc<Mutex<Vec<(LogRecord, InstrumentationLibrary)>>>,
+    }
+
+    impl LogExporterThatRequiresTokioSpawn {
+        /// Creates a new instance of `LogExporterThatRequiresTokioSpawn`.
+        pub(crate) fn new() -> Self {
+            LogExporterThatRequiresTokioSpawn {
+                logs: Arc::new(Mutex::new(Vec::new())),
+            }
+        }
+
+        /// Returns the number of logs stored in the exporter.
+        pub(crate) async fn len(&self) -> usize {
+            let logs = self.logs.lock().unwrap();
+            logs.len()
+        }
+    }
+
+    use tokio::time::sleep;
+
+    #[async_trait::async_trait]
+    impl LogExporter for LogExporterThatRequiresTokioSpawn {
+        async fn export(&mut self, batch: LogBatch<'_>) -> LogResult<()> {
+            // Simulate minimal dependency on tokio by sleeping for a short duration
+            sleep(Duration::from_millis(50)).await;
```

Comment:
```suggestion
            tokio::time::sleep(Duration::from_millis(50)).await;
```

I think this makes it easy to spot that this is tokio sleep